modules/commandModule/commandWorker_flight.py

```diff
@@ -15,4 +19,34 @@ def flight_command_worker(pipelineIn, pipelineOut):
 
 	#TODO: Write logic to get POGI
 
-	
\ No newline at end of file
+
+def pogi_command_worker(pause, exitRequest, pipelineOut):
+    """
+    Producer worker function to return data currently in the POGI file
+
+    Parameters
+    ----------
+    pause : multiprocessing.Lock
+        a lock shared between worker processes
+    exitRequest : multiprocessing.Queue
+        a queue that determines when the worker process stops
+    pipelineOut : multiprocessing.Queue
+        output pipeline for pogiData
+    """
+
+    print("Reading POGI data")
+
+    while True:
+        pause.acquire()
+        pause.release()
+
+        # get the pogi data
+        pogiData = read_pogi()
+
+        # output to pipeline
+        # note: no null checking required since CommandModule returns None for fields that are unreadable/inaccesible
+        #       if all fields are None, then pogiData will be a dict with all elements as {'example_key' : None}
+        pipelineOut.put(pogiData)
```

Comment:
Using what I mentioned above, add an if statement here that checks if `changed` is `True` before putting the pogiData in the pipeline.